ts-tests/.env.example

```diff
@@ -0,0 +1 @@
+NODE_ENV=<`development` or `staging` or `test` or `ci`>
```

Comment:
How can you guarantee the NODE_ENV on CI server is ci? Should we set it up manually?

Reply:
https://github.com/litentry/litentry-parachain/blob/a1763aff06c6bf22cb4e0609bc664c469ee811e1/ts-tests/scripts/run-test.sh#L12